src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -65,6 +65,14 @@
     <Compile Include="$(MessagePackRoot)SequencePool.cs" LinkBase="BlazorPack\MessagePack" />
     <Compile Include="$(MessagePackRoot)SequenceReader.cs" LinkBase="BlazorPack\MessagePack" />
     <Compile Include="$(MessagePackRoot)SequenceReaderExtensions.cs" LinkBase="BlazorPack\MessagePack" />
+
+    <!-- Shared descriptor infrastructure with MVC -->
+    <Compile Include="$(RepoRoot)src\Shared\Components\ComponentDescriptor.cs" LinkBase="Descriptors" />
+    <Compile Include="$(RepoRoot)src\Shared\Components\ComponentDescriptorInstance.cs" LinkBase="Descriptors" />
+    <Compile Include="$(RepoRoot)src\Shared\Components\ComponentDescriptorSerializer.cs" LinkBase="Descriptors" />
+    <Compile Include="$(RepoRoot)src\Shared\Components\ComponentDescriptorSerializer.Deserialize.cs" LinkBase="Descriptors" />
+    <Compile Include="$(RepoRoot)src\Shared\Components\ComponentMarker.cs" LinkBase="Descriptors" />
```

Comment:
> LinkBase="Descriptors"

Do we need this? We try to have folders match namespaces.

Reply:
I removed these from here. Most of these things are in Microsoft.AspNetCore.Components (as I didn't want to add .Server because it's referenced by MVC too). I've removed the LinkBase